content/blog/2020-02-17-a-public-reddit-dataset.md

```diff
@@ -110,7 +110,7 @@ you'll need to [install DVC](https://dvc.org/doc/install); one of the simplest
 ways is `pip install dvc`.
 
 Say you have a directory on your local machine where you plan to build some
-analysis scripts. Simply run
+analysis scripts. You can run the command:
```

Comment:
Again I don't see "simply" as condescending here. You could just replace it with "Just" to be sure though, instead of adding several more words.

Reply:
Agreed, I feel let it be like that itself, because - just is also a condescending word instance

Reply:
I see you went for "You run" — doesn't sound very good. Please use "You can run" for now but this is also in the blog so let's see what @andronovhopf thinks.

p.s. please don't resolve conversations if you're not 100% sure they're resolved because the Github interface is not the best for collapsing and expanding them. Thanks